cmd/soci/commands/index/info.go

```diff
@@ -63,7 +65,21 @@ var infoCommand = cli.Command{
 		}
 		defer reader.Close()
 
-		_, err = io.Copy(os.Stdout, reader)
+		b, err := io.ReadAll(reader)
+		if err != nil {
+			return err
+		}
+
+		err = prettyPrintJSON(b)
 		return err
 	},
 }
+
+func prettyPrintJSON(b []byte) error {
+	var prettyJSON bytes.Buffer
+	if err := json.Indent(&prettyJSON, []byte(b), "", "  "); err != nil {
```

Comment:
Oh, nice! This is much nicer than my suggestion to deserialize into `map[string]interface{}`.